app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -34,5 +34,11 @@ def t(key, interpolations = {})
     end
     helper HelperOverride
 
+    # Adds the 'My Works' breadcrumb; we only want this for logged-in users
+    # overrides https://github.com/samvera/sufia/blob/v7.3.1/app/controllers/concerns/sufia/works_controller_behavior.rb#L93
+    def add_breadcrumb_for_controller
+      super if current_ability.current_user.staff?
+    end
```

Comment:
Yes, it's the same right now. But it could change in the future. 

While "staff?" and "logged in" are the same thing right now, in some cases we _really_ want to be checking for "staff" (if there was a logged in user that was not staff, they should not be included). 

But in this case I think we _really_ want to check for "logged in" -- if there's a logged in user that's not staff, they _should_ still get the breadcrumbs.

Do you happen to know if there's an easy way to distinguish that dummy user?  If there is, I _think_ that would be better semantically. But not really a big deal, if it's a confusing pain this is probably good enough for now -- hypothetically adding non-staff logged in users in the future would prob break lots of things already.